src/scopes/activityReport/participants.js

```diff
@@ -0,0 +1,29 @@
+import filterArray from './utils';
+import {
+  ALL_PARTICIPANTS,
+} from '../../constants';
+
+function onlyValidParticipants(query) {
+  if (!Array.isArray(query)) {
+    return [query].filter((reason) => ALL_PARTICIPANTS.includes(reason));
+  }
```

Comment:
😃 might want to change that iterator from ```reason``` to ```participant```

Reply:
Nice catch.